README.md

```diff
@@ -23,7 +23,7 @@ Optional:
 * `aliases` — list of alias domains
 * `global` — set a global redirect or archive for all paths
 * `css` — a css class which determines the logo and brand colour used on 404/410 pages
-* `options` — used to list significant querystrings for canonicalisation like this: `--query-string first:second:third`. A significant querystring is one which on the old website changes the content in a meaningful way - which we might therefore need to map to a different place.
+* `options` — used to list significant querystrings for canonicalisation like this: `--query-string first:second:third`. A significant querystring parameter is one which on the old website changes the content in a meaningful way - which we might therefore need to map to a different place. **Query string parameters should be lower-case; upper case confuses the canonicalisation**
```

Comment:
Could we be more specific here? Eg 'Query string parameters should be specified in lower-case here. Any which include upper-case letters will not be preserved in mappings.'

It would be good to validate this in the build as we do for hostnames, and also prevent upper-case query params from being imported into Transition.